controllers/fetchers/jsonnet_fetcher.go

```diff
@@ -0,0 +1,15 @@
+package fetchers
+
+import (
+	"github.com/google/go-jsonnet"
+	"github.com/grafana-operator/grafana-operator-experimental/api/v1beta1"
+)
+
+func FetchJsonnet(dashboard *v1beta1.GrafanaDashboard) (string, error) {
+
+	vm := jsonnet.MakeVM()
```

Comment:
I wonder if there's a way to have a global VM that we reset after every import?

Reply:
Hmmm, that's not a bad idea.
Do you think it's a safe assumption to make that if one dashboard is using jsonnet, that the remainder will be as well?

Reply:
Not sure, thinking about it again, maybe its not expensive to recreate a fresh VM and this way we also provide isolation between dependencies used by dashboards.

Reply:
Well, I guess we can refactor to a more permanent VM if there's a big use case for jsonnet, but for now, wasteful to provision it for a single dashboard type if there's a mixture of dashboard types.

Albeit, the person using jsonnet, would probably use jsonnet for all their dashboards.